config/rbac/role.yaml

```diff
@@ -77,3 +77,9 @@ rules:
   - get
   - patch
   - update
+- apiGroups:
+  - appstudio.redhat.com
```

Comment:
I feel it should be `- [""]` for `secrets`

Reply:
I think you're right but it should be `- ""` instead (based on this [example](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#role-example)), right?

Reply:
Yes, without `[]`, thanks!

Reply:
This has been fixed in 05567bfbaa69715ecc5dacf23c54b5cf377ac456.

Reply:
@hongweiliu17 please resolve this convo if you agree with the fix. Thanks!

Reply:
Thanks!